src/library/trackset/playlist/playlist.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "library/trackset/playlist/playlistid.h"
+#include "util/db/dbnamedentity.h"
+
+class Playlist : public DbNamedEntity<PlaylistId> {
```

Comment:
Is this new class used? Can this be just a typedef? The name is IMHO ambiguous.

Reply:
Yes, `PlaylistSummary` class is derived from this class.

Well, I was trying to follow and replicate how `CrateStorage` is implemented. The idea is that one day `PlaylistStorage`, which is introduced with this PR, will replace `PlaylistDAO`, just like `CrateStorage` replaced `CrateDAO`.